.github/workflows/check.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        GO_VERSION: [ "1.16","1.17","1.18" ]
+        GO_VERSION: [ "1.16","1.17","1.18","1.19","1.20","1.21"" ]
```

Comment:
Shouldn't we only stick with the supported versions of go 🤔 ?

Reply:
I only added versions. I didn't want to remove anything. :)